lib/jvertica/error.rb

```diff
@@ -0,0 +1,7 @@
+class Jvertica
+  class Error < StandardError; end
+  class ConnectionError < Error; end
```

Comment:
Superclass should be`Jvertica::Error` rather than `StandardError`. This enables to rescue like

```
rescue Jvertica::Error => e
end
```